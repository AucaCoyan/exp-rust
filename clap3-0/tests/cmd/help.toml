bin.name = "espanso"
args = "help"
