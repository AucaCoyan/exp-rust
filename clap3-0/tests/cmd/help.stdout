espanso 

USAGE:
    espanso.exe [OPTIONS] [name] [SUBCOMMAND]

ARGS:
    <name>    Optional name to operate on

OPTIONS:
    -c, --config <FILE>    Sets a custom config file
    -d, --debug            Turn debugging information on
    -h, --help             Print help information

SUBCOMMANDS:
    help    Print this message or the help of the given subcommand(s)
    test    does testing things
